dev_guide/storage_classes.adoc

```diff
@@ -0,0 +1,26 @@
+[[dev-guide-storage-classes]]
+= Storage Classes
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+
+The `StorageClass` resource object describes and classifies storage that can be
+requested, as well as provides a means for passing parameters for dynamically
+provisioned storage on demand. `StorageClass` objects can also serve as a
+management mechanism for controlling different levels of storage and access to
+the storage. Cluster administrators (users with `cluster-admin` privileges) or
+storage administrators (users with `storage-admin` privileges) define and create
+the `StorageClass` objects that users can request without needing any intimate
+knowledge about the underlying storage volume sources.
+
+In {product-title}, the storage class is configured and a single option is
+available to the user based on the underlying cloud provider.
```

Comment:
@abhgupta @damemi should we have cloud/storage providers actually listed by name there? Thanks!